src/flows/components/Sidebar.tsx

```diff
@@ -262,7 +318,9 @@ const Sidebar: FC = () => {
                 event('Notebook Nav: Called Action', {menu: title})
                 // eslint-disable-next-line no-extra-semi
                 ;(action as ControlAction).action()
-                hide()
+                if (action.title !== 'Download As PDF') {
```

Comment:
this feels wrong, giving the sidebar menu controller a concept of the PDF function. before figuring out how to move this, what production functionality provoked this addition?